src/stabilization_report_template.md

```diff
@@ -0,0 +1,54 @@
+# Stabilization report template
+
+> **What is this?** This is a template to use for [stabilization reports](./stabilization_guide.md). It consists of a series of questions that aim to provide the information most commonly needed and to help reviewers be more likely to identify potential problems up front. Not all parts of the template will apply to all stabilizations. Feel free to put N/A if a question doesn't seem to apply to your case.
```

Comment:
This should mention that it is a stabilization for *language* features, at least in passing. Similar to the big note at the top of stabilization_guide.md itself.

With the current wording it is way to easy to stumble upon this link, assume it applies to T-libs stabilization as well, and be very confused.

Reply:
Added clarification that this is for language features